src/Ownership/Commands/ApproveOwnership.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Ownership\Commands;
+
+use CultuurNet\UDB3\Model\ValueObject\Identity\UserId;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+
+final class ApproveOwnership
+{
+    private UUID $id;
+    private UserId $approverId;
+
+    public function __construct(UUID $id, UserId $requesterId)
```

Comment:
Variable and property name not in sync? Copy-paste mistake.

Reply:
I changed this to be consistent and always use requester, see https://github.com/cultuurnet/udb3-backend/pull/1668/commits/eb50b40019a185d92412605f64711f8391b21f9e

This is also the property name of the JSON LD projection